GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -246,17 +247,15 @@ const float *Mesh::getBoundingSphereInfo() {
 }
 
 // generate vertex array object
-void Mesh::generateVAO(Material::ShaderType key) {
+void Mesh::generateVAO() {
 #if _GVRF_USE_GLES3_
+
```

Comment:
formatting is off.  maybe you have tabs rather than spaces?

http://gearvrf.org/bin/view/GearVRF/GearVRfWikiGetInvolved#CodeGuide